hathor/transaction/base_transaction.py

```diff
@@ -1037,6 +1037,18 @@ def clone(self) -> 'BaseTransaction':
     def get_token_uid(self, index: int) -> bytes:
         raise NotImplementedError
 
+    @abstractmethod
+    def get_requested_from_height(self) -> int:
```

Comment:
It seems completely related to the operation of the index. Why don't you put this inside the index instead of here?

Reply:
It's more ergonomic to have this on the transaction. But I can move this to an isolated standalone function somewhere on the indexes module (maybe `hathor.indexes.deps_index`).

Reply:
I had missed this, I just updated to move `get_requested_from_height` to a standalone function in the `hathor.indexes.deps_index`. What do you think?

Reply:
Seems good. Thanks!